_test/.phpunit.result.cache

```diff
@@ -0,0 +1 @@
 set #11";d:0.018;s:89:"dokuwiki\plugin\config\test\Setting\SettingOnoffTest::testShouldBeSaved with data set #12";d:0.014;s:89:"dokuwiki\plugin\config\test\Setting\SettingOnoffTest::testShouldBeSaved with data set #13";d:0.024;s:89:"dokuwiki\plugin\config\test\Setting\SettingOnoffTest::testShouldBeSaved with data set #14";d:0.015;s:89:"dokuwiki\plugin\config\test\Setting\SettingOnoffTest::testShouldBeSaved with data set #15";d:0.016;s:89:"dokuwiki\plugin\config\test\Setting\SettingOnoffTest::testShouldBeSaved with data set #16";d:0.015;s:89:"dokuwiki\plugin\config\test\Setting\SettingOnoffTest::testShouldBeSaved with data set #17";d:0.016;s:71:"dokuwiki\plugin\config\test\Setting\SettingOnoffTest::testInitialBasics";d:0.013;s:75:"dokuwiki\plugin\config\test\Setting\SettingOnoffTest::testShouldHaveDefault";d:0.016;s:67:"dokuwiki\plugin\config\test\Setting\SettingOnoffTest::testPrettyKey";d:0.015;s:62:"dokuwiki\plugin\config\test\Setting\SettingOnoffTest::testType";d:0.017;s:65:"dokuwiki\plugin\config\test\Setting\SettingOnoffTest::testCaution";d:0.014;s:79:"dokuwiki\plugin\config\test\Setting\SettingStringTest::testOut with data set #0";d:0.015;s:79:"dokuwiki\plugin\config\test\Setting\SettingStringTest::testOut with data set #1";d:0.019;s:89:"dokuwiki\plugin\config\test\Setting\SettingStringTest::testShouldBeSaved with data set #0";d:0.018;s:89:"dokuwiki\plugin\config\test\Setting\SettingStringTest::testShouldBeSaved with data set #1";d:0.016;s:89:"dokuwiki\plugin\config\test\Setting\SettingStringTest::testShouldBeSaved with data set #2";d:0.016;s:72:"dokuwiki\plugin\config\test\Setting\SettingStringTest::testInitialBasics";d:0.014;s:76:"dokuwiki\plugin\config\test\Setting\SettingStringTest::testShouldHaveDefault";d:0.015;s:68:"dokuwiki\plugin\config\test\Setting\SettingStringTest::testPrettyKey";d:0.015;s:63:"dokuwiki\plugin\config\test\Setting\SettingStringTest::testType";d:0.016;s:66:"dokuwiki\plugin\config\test\Setting\SettingStringTest::testCaution";d:0.014;s:73:"dokuwiki\plugin\config\test\Setting\SettingTest::testOut with data set #0";d:0.015;s:73:"dokuwiki\plugin\config\test\Setting\SettingTest::testOut with data set #1";d:0.017;s:83:"dokuwiki\plugin\config\test\Setting\SettingTest::testShouldBeSaved with data set #0";d:0.014;s:83:"dokuwiki\plugin\config\test\Setting\SettingTest::testShouldBeSaved with data set #1";d:0.016;s:83:"dokuwiki\plugin\config\test\Setting\SettingTest::testShouldBeSaved with data set #2";d:0.015;s:66:"dokuwiki\plugin\config\test\Setting\SettingTest::testInitialBasics";d:0.017;s:70:"dokuwiki\plugin\config\test\Setting\SettingTest::testShouldHaveDefault";d:0.016;s:62:"dokuwiki\plugin\config\test\Setting\SettingTest::testPrettyKey";d:0.016;s:57:"dokuwiki\plugin\config\test\Setting\SettingTest::testType";d:0.015;s:60:"dokuwiki\plugin\config\test\Setting\SettingTest::testCaution";d:0.018;s:48:"dokuwiki\plugin\config\test\WriterTest::testSave";d:0.018;s:49:"dokuwiki\plugin\config\test\WriterTest::testTouch";d:1.068;}}}
```

Comment:
I guess this is an autogenerated file, please exclude it from the PR.

Reply:
Done.